gradio/chat_interface.py

```diff
@@ -182,10 +186,18 @@ def __init__(
                 Markdown(description)
 
             if chatbot:
+                if self.msg_format != chatbot.msg_format:
+                    raise ValueError(
```

Comment:
I can imagine users passing in a custom `gr.Chatbot` but forgetting to set `msg_format` to to "messages" so alternatively, we could just raise a warning when they mismatch and say that the `msg_format` to `ChatInterface` will be used